### PR TITLE
Improvement glossary UI in the front end

### PR DESCRIPTION
**Pull request by skyepurchase**

### CODE REVIEW

File: src/app/components/pages/Glossary.tsx [hunk 1]
```diff
@@ -278,13 +279,15 @@ export const Glossary = () => {
                     <Row className="no-print">
                         {isPhy && <Col className="mt-3 mt-md-0">
                             <Label for='subject-select' className='sr-only'>Subject</Label>
-                            <StyledSelect inputId="subject-select"
-                                options={ subjects.map(e => ({ value: e.id, label: e.title})) }
-                                value={filterSubject ? ({value: filterSubject.id, label: filterSubject.title }) : undefined}
+                            <Select inputId="subject-select"
```
Comment (jacbn) [inline src/app/components/pages/Glossary.tsx:282]:
```suggestion
                            <Select inputId="subject-select" className="glossary-select"
```

File: src/scss/phy/glossary.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
+@import "../common/glossary";
+
+.biology {
```
Comment (jacbn) [inline src/scss/phy/glossary.scss:3]:
```suggestion
.glossary-select {
    &.biology {
```

File: src/scss/phy/glossary.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    .select__control {
+        background-color: $phy_biology !important;
+    }
+
+    .select__single-value {
+        color: #fff;
+    }
+}
+
+.chemistry {
```
Comment (jacbn) [inline src/scss/phy/glossary.scss:13]: these three will need the same treatment

**Review by jacbn — APPROVED** [review]
Nice, works great. I'm a bit worried though about defining `.physics`/`.maths`/etc as global properties, as they get used in a few other places in the code. It's _probably_ okay here as you have another `.select` after anyway, but in the off chance someone changes this / adds a select elsewhere it's worth wrapping them in some tag that will definitely only get used by the glossary.

File: src/app/components/pages/Glossary.tsx (hunk 1 above)
Comment (skyepurchase) [reply src/app/components/pages/Glossary.tsx:282]: There already exists a className prop so will put it in that instead

**Pull request merged**